civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -95,13 +95,11 @@
 // Custom CiviHR Theme CSS
 // --------------------------------------------------
 @import "base/custom";
-
+@import "base/_temp_custom";
```

Comment:
also, when you `@import` a partial, you don't need to specify the `_` (see other imports in the same file)